tidysic/main.py

```diff
@@ -1,20 +1,23 @@
 from tidysic.argparser import create_parser
-from tidysic.tidysic import organize
+from tidysic.tidysic import Tidysic
+
+from tidysic.ordering import Ordering, OrderingStep
+from tidysic.tag import Tag
+from tidysic.formatted_string import FormattedString
 
 
 def run():
     parser = create_parser()
     args = parser.parse_args()
 
-    organize(
-        args.source,
-        args.target,
-        args.with_album,
-        args.with_clutter,
-        args.guess,
-        args.dry_run,
-        args.verbose
-    )
+    tidysic = Tidysic(args)
+    if not args["with_album"]:
+        tidysic.ordering = Ordering(
+            OrderingStep(Tag.Artist, FormattedString("{{artist}}")),
+            OrderingStep(Tag.Title, FormattedString("{{track}. }{{title}}"))
+        )
+
+    tidysic.organize()
 
 
 if __name__ == '__main__':
```

Comment:
It doesn't we're having the convention to add an empty line between a conditional statement and its nested statement.

Reply:
We don't have anything resembling convention around here (except for the fact that we try to prefer single quotes to double ones).